tutorial.md

```diff
@@ -194,7 +194,7 @@ SiteConfig:
     - SiteConfigExtension
 ```
 
-Because we changed the config, we have to flush the cache. Build the database using `dev/build?flush`. You should see some new fields.
+Because we changed the config, we have to flush the cache. Build the database using `dev/build?flush=1`. You should see some new fields.
```

Comment:
No you don't, but until `=all` is deprecated it's better to be teaching newbies that there are actually options for it, IMHO